timeline_logger/manager.py

```diff
@@ -0,0 +1,22 @@
+from django.contrib.contenttypes.models import ContentType
+from django.db.models import Manager, Model
+
+
+class TimelineLogManager(Manager):
+    def for_object(self, obj: Model):
+        assert isinstance(obj, Model)
+
+        app_label = obj._meta.app_label
+        model_name = obj._meta.model_name
+
+        try:
+            content_type = ContentType.objects.get(
+                app_label=app_label, model=model_name
+            )
+        except ContentType.DoesNotExist:
+            return self.none()
```

Comment:
should just be:

```py
content_type = ContentType.objects.get_for_model(obj)
```

See https://docs.djangoproject.com/en/4.2/ref/contrib/contenttypes/#django.contrib.contenttypes.models.ContentTypeManager.get_for_model